data-tools.js

```diff
@@ -282,8 +282,13 @@ export default class TidepoolDataTools {
     return JSONStream.parse('*');
   }
 
-  static tidepoolProcessor(processorConfig = {}) {
+  static tidepoolProcessor(processorConfig = {}, filteredType = false) {
```

Comment:
why not use an empty array instead of "false" by default? filteredType is suposed to be used as an array